backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -75,4 +108,19 @@ trait WorkflowBackendActor extends Actor with ActorLogging {
     * Abort a running job.
     */
   def abort(jobKey: BackendJobDescriptorKey): Future[AbortResponse]
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  def validate(): Future[InitializationResponse]
+
+  /**
+    * A call which happens before anything else runs
+    */
+  def beforeAll(): Future[Unit]
```

Comment:
One use case that may happen is when the backend developer does not deal well with exceptions, so if an unhandled exception happen, SWA should be able to recover from that situation and fail gracefully.